lib/ansible/modules/windows/win_group_membership.py

```diff
@@ -27,9 +24,11 @@
   members:
     description:
       - A list of members to ensure are present/absent from the group.
-      - Accepts local users as username, .\username, and SERVERNAME\username.
+      - Accepts local users as .\username, and SERVERNAME\username.
```

Comment:
`username` should still work, the only case where it wouldn't AFAIC is if the username is a valid sid string. If that is the case they would have to do `.\` and they probably deserve their punishment for doing something so silly.

Reply:
Hey @jborean93, I decided to move the description of the `username` format down to its own lines (30-31) considering how local, domain and service users can now utilize this format, and it required a bit of explanation around the domain lookup favoring.  Happy to change things around in the docs if you think it'd be clearer expressed differently.

For the tests, we're still testing plain username without the host prefix (among others), however I changed this to target a test-generated local account rather than `Guest`.  Mostly, to be in alignment with the strategy now used for the local administrator account, and to avoid any conflation with the new domain precedence (since guest exists by default both locally and in domain environments).

Reply:
No worries, was just making sure it wasn't done for incompatibility reasons. Changes look good and I was able to replicate/prove this fixes https://github.com/ansible/ansible/issues/40649 as part of my testing.